deno.json

```diff
@@ -1,6 +1,6 @@
 {
   "tasks": {
-    "build": "cp LICENSE js/LICENSE && RUSTFLAGS=--cfg=web_sys_unstable_apis deno run -A jsr:@deno/wasmbuild@0.17.1 --out js",
+    "build": "cp LICENSE js/LICENSE && RUSTFLAGS=--cfg=web_sys_unstable_apis deno run -A jsr:@deno/wasmbuild@0.17.1 --all-features --out js",
```

Comment:
@dsherret @lucacasonato FYI I propose to enable all checksums for the wasm build